enhancements/monitoring/replace-prometheus-adapter-with-metrics-server.md

```diff
@@ -0,0 +1,189 @@
+---
+title: replace-prometheus-adapter-with-metrics-server
+authors:
+  - "@slashpai"
+reviewers:
+  - "@simonpasquier"
+  - "@dgrisonnet"
+  - "@openshift/openshift-team-monitoring"
+approvers: 
+  - "@simonpasquier"
+  - "@dgrisonnet"
+  - "@openshift/openshift-team-monitoring"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-06-20
+last-updated: 2023-06-20
+tracking-link:
+  - https://issues.redhat.com/browse/MON-3153
+---
+
+# Replace prometheus-adapter with metrics-server
+
+## Summary
+
+This proposal suggests replacing the [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) currently used for monitoring resource metrics in OpenShift with the [metrics-server](https://github.com/kubernetes-sigs/metrics-server).
+
+## Motivation
+
+For autoscaling OpenShift needs a resource metrics implementation. Currently this depends on CMO's Prometheus stack.
+
+Reasons to consider to moving to metrics-server are:
+
+1. Initially we planned using [prometheus-adapter](https://github.com/kubernetes-sigs/prometheus-adapter) for custom metrics API too but the direction has now shifted towards KEDA [OCPNODE-708]((https://issues.redhat.com/browse/OCPNODE-708)) so there's little incentive from the OpenShift monitoring team's perspective to continue investing into prometheus-adapter.
```

Comment:
```suggestion
1. **Simplified Monitoring Architecture** By replacing prometheus-adapter with metrics-server, we can reduce the complexity and maintenance overhead of the monitoring system in OpenShift since `metrics-server` does not rely on Prometheus.
```